apps/api/env/.env.testnet

```
@@ -1,2 +1,3 @@
+REST_API_NODE_URL=https://testnetapi.akashnet.net
 RPC_NODE_ENDPOINT=https://testnetrpc.akashnet.net:443
-DEPLOYMENT_GRANT_DENOM=uakt
\ No newline at end of file
+DEPLOYMENT_GRANT_DENOM=uact
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEPLOYMENT_GRANT_DENOM key should go before the REST_API_NODE_URL key

(UnorderedKey)